src/plugin/featureaction/featureactionmanager.js

```diff
@@ -168,6 +168,25 @@ plugin.im.action.feature.Manager.prototype.onSourceRemoved_ = function(event) {
   }
 };
 
+/**
+ * @param {os.source.Request} source
+ * @return {boolean}
+ */
+plugin.im.action.feature.Manager.prototype.hasActiveActions = function(source) {
+  var featureActions = this.getActionEntries(source.getId());
+  var hasActiveActions = false;
+
+  if (featureActions.length > 0) {
+    for (var i = 0; i < featureActions.length; i++) {
+      if (featureActions[i].enabled) {
+        hasActiveActions = true;
+      }
+    }
```

Comment:
This loop can be more cleanly written using `featureActions.some()`. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some